### ISSUE TITLE
Specify Markdown Style

### ISSUE THREAD

**Issue by meichthys** [mattrob33/christian-projects#2]
Since we are using markdown in the main readme, perhaps we should specify the markdown format/linter we want to use/recommend in order to enforce consistency? I think VSCode uses [markdownlint](https://github.com/markdownlint/markdownlint) by default.

**Comment by mattrob33** [comment]
Good idea. This would be a great use case for using a [Github Action](https://docs.github.com/en/actions) to automatically run a linter like [markdownlint](https://github.com/markdownlint/markdownlint) on all PRs. I'll add a `help wanted` label if you or someone else want to look into this before I get a chance to do so.

**Comment by meichthys** [comment]
Oo i like the github action idea. I forgot that actions are now free for public repo!

**Comment by mattrob33** [comment]
Yeah they're pretty sweet. I've set up a few on different projects and could help give some guidance and answers questions, but I haven't done Markdown linting (or actually any linting) with them before, so the specifics would take some research time that I don't have at the moment.

**Comment by NonlinearFruit** [comment]
If the goal is to be similar to an awesome list, they have a simple linter ([source](https://github.com/sindresorhus/awesome-lint))

**Comment by StaelTchinda** [comment]
I tried to implement it using [markdownlint-cli2-action](https://github.com/marketplace/actions/markdownlint-cli2-action). My approach is on my [forked project](https://github.com/StaelTchinda/christian-projects/pull/1).

However, it does not work. I tried to set the contribution rules as custom rules using JavaScript (see [code](config/readme.markdownlint.jsonc), but it seems like code dependencies cannot be added dynamically to Github Actions(see [issue](https://github.com/DavidAnson/markdownlint-cli2-action/issues/134#issuecomment-1986468777)).

Any suggestion on what to do and how ?
